src/components/richText/richText.tsx

```diff
@@ -39,22 +39,43 @@ const externalHyperlinkRenderer = (node: any): ReactNode => {
     return <a href={` ${ location } `} target="_blank" rel="noopener">{ text }</a>;
 };
 
+const inlineEmbeddedEntryRenderer = (node: any): ReactNode => {
+    try {
+        if (node.data.target.sys.contentType.sys.id === 'partner' && node.data.target.fields.partnerType['en-US'] === 'Lunch Venue') {
```

Comment:
seems odd to have `'en-US'` in this hard coded so many times, or at all.

Reply:
fair